services/github-microservice.ts

```diff
@@ -126,4 +135,12 @@ export default class GithubMicroService {
                    return null;
                  })
   }
+  static async getMergeProposalIssue(issueId: string, MergeId: string) {
```

Comment:
note: this function can return null but the code never accounts for that, this will bring problems later on. account for this null return when you consume the value from this method.